Plugins/Public/base_plugin/PlayerCommands.cpp

```diff
@@ -2031,109 +2138,109 @@ namespace PlayerCommands
 					}
-					break;
+					else
+					{
+						ConPrint(L"Trade Lane too close, distance: %um", static_cast<uint>(distance));
+					}
+					return false;
 				}
-				case OBJ_JUMP_GATE:
-				case OBJ_JUMP_HOLE:
+				break;
+			}
+			case OBJ_JUMP_GATE:
+			case OBJ_JUMP_HOLE:
+			{
+				if (distance < minJumpDistance)
 				{
-					if (distance < minJumpDistance)
-					{
-						uint idsName = solar->get_name();
-						if (!idsName) idsName = solar->get_archetype()->iIdsName;
+					uint idsName = solar->get_name();
+					if (!idsName) idsName = solar->get_archetype()->iIdsName;
 
-						if (client)
-						{
-							PrintUserCmdText(client, L"%ls too close. Current: %um, Minimum distance: %um", HkGetWStringFromIDS(idsName).c_str(), static_cast<uint>(distance), static_cast<uint>(minJumpDistance));
-						}
-						else
-						{
-							ConPrint(L"Base too close to %ls, distance: %um", HkGetWStringFromIDS(idsName).c_str(), static_cast<uint>(distance));
-						}
-						return false;
+					if (client)
+					{
+						PrintUserCmdText(client, L"%ls too close. Current: %um, Minimum distance: %um", HkGetWStringFromIDS(idsName).c_str(), static_cast<uint>(distance), static_cast<uint>(minJumpDistance));
 					}
-					break;
+					else
+					{
+						ConPrint(L"Base too close to %ls, distance: %um", HkGetWStringFromIDS(idsName).c_str(), static_cast<uint>(distance));
+					}
+					return false;
 				}
-				case OBJ_SATELLITE:
-				case OBJ_WEAPONS_PLATFORM:
-				case OBJ_DESTROYABLE_DEPOT:
-				case OBJ_NON_TARGETABLE:
-				case OBJ_MISSION_SATELLITE:
+				break;
+			}
+			case OBJ_SATELLITE:
+			case OBJ_WEAPONS_PLATFORM:
+			case OBJ_DESTROYABLE_DEPOT:
+			case OBJ_NON_TARGETABLE:
+			case OBJ_MISSION_SATELLITE:
+			{
+				if (distance < minDistanceMisc)
 				{
-					if (distance < minDistanceMisc)
+					uint idsName = solar->get_name();
+					if (!idsName) idsName = solar->get_archetype()->iIdsName;
+					if (client)
 					{
-						uint idsName = solar->get_name();
-						if (!idsName) idsName = solar->get_archetype()->iIdsName;
-						if (client)
-						{
-							PrintUserCmdText(client, L"%ls too close. Current: %um, Minimum distance: %um", HkGetWStringFromIDS(idsName).c_str(), static_cast<uint>(distance), static_cast<uint>(minDistanceMisc));
-						}
-						else
-						{
-							ConPrint(L"Base too close to %ls, distance: %um", HkGetWStringFromIDS(idsName).c_str(), static_cast<uint>(distance));
-						}
-						return false;
+						PrintUserCmdText(client, L"%ls too close. Current: %um, Minimum distance: %um", HkGetWStringFromIDS(idsName).c_str(), static_cast<uint>(distance), static_cast<uint>(minDistanceMisc));
 					}
-					break;
+					else
+					{
+						ConPrint(L"Base too close to %ls, distance: %um", HkGetWStringFromIDS(idsName).c_str(), static_cast<uint>(distance));
+					}
+					return false;
 				}
+				break;
+			}

```

Comment:
what's going on here with the closing brackets/indentation?

Reply:
First one is a case block closing, second is the entire switch block closing